qemu/tests/virtual_nic_stress.py

```diff
@@ -31,9 +32,15 @@ def flood_ping(session, host_ip, os_type="linux"):
         logging.info("Flood ping for %s minutes" % flood_minutes)
         # TODO: windows guest doesn't have proper flood ping like linux,
         # will consider how to implement it for windows later
```

Comment:
Perhaps "TODO" comment here can also be deleted like above in cfg?

Reply:
Yes, modified, thanks for reminder